src/reducers/entities/channels.js

```diff
@@ -492,6 +501,23 @@ function totalCount(state = 0, action) {
     }
 }
 
+export function manuallyUnread(state = {}, action) {
+    switch (action.type) {
+    case ChannelTypes.REMOVE_MANUALLY_UNREAD: {
+        // use destructuring to both remove the element and create the new state if it exist.
+        const {[action.data.channelId]: removeThis, ...newState} = state;
+        if (removeThis) {
+            return newState;
+        }
+        break;
+    }
+    case ChannelTypes.POST_UNREAD_SUCCESS: {
+        return {...state, [action.data.channelId]: true};
+    }
+    }
+    return state;
```

Comment:
Similarly, I'd recommend that we stick to using a default case in this switch statement like we do elsewhere.